src/8_logger.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+goog.provide('Logger');
+// jscs:disable validateIndentation
+/**
+ * @constructor
+ */
+
+Logger = function() {
+  /**
+   * @private
+   * @type {string}
+   */
+  this.level_ = 'error';
+  this.levelsOrdered = [ 'verbose', 'info', 'warn', 'error', 'none' ];
+};
+
+/**
+ * Sets the logging level.
+ * @param {string} level - The logging level to set.
+ */
+Logger.prototype.setLevel = function(level) {
+  if (this.levelsOrdered.indexOf(level) !== -1) {
+    this.level_ = level;
+  }
+  else {
+    console.error(`Invalid log level: ${level}`);
```

Comment:
If the level is left empty, will this result in no logs? Would this stop existing logs out there from printing if this were released as the default prod package?

Reply:
Scratch that, they'd have to use the wrapper first.